### DISCUSSION TITLE
500 Server Error

### DISCUSSION THREAD

**Discussion by amiria703** [yhype/yhype-feedback#19] [General]
I get 500 when trying to sign up or log in.
Unfortunately, this issue still persists. :(

**Comment by antonkomarev** [comment]
@amiria703 Try again, please.

**Reply by amiria703** [reply]
Now it works! TYVM.

**Comment by antonkomarev** [comment]
We found 2 issues and resolved them. Should not return 500 now. 2 days without issues. Thanks for reporting!
